corehq/apps/app_manager/templates/app_manager/module_view_report.html

```diff
@@ -57,6 +106,60 @@
                         <td>
                             <select data-bind="options: availableReportIds, optionsText: $root.defaultReportTitle, value: reportId"></select>
+                            <table class="table">
+                                <thead>
+                                    <tr>
+                                        <th>{% trans 'Filter Name' %}</th>
+                                        <th>{% trans 'Type' %}</th>
+                                        <th></th>
+                                    </tr>
+                                </thead>
+                                <tbody data-bind="foreach: { data: filterConfig.selectedFilterStructure, as: 'filter' }">
+                                    <tr>
+                                        <th data-bind="text: filter.display"></th>
+                                        <th>
+                                            <select data-bind="
+                                                options: report.filterConfig.filterDocTypes,
+                                                value: selectedValue.doc_type
+                                            ">
+                                            </select>
+                                        </th>
+                                        <th>
+                                            <div data-bind="visible: filter.selectedValue.doc_type() == 'AutoFilter'">
+                                                <select data-bind="
+                                                    options: report.filterConfig.autoFilterTypes,
+                                                    value: filter.selectedValue.filter_type
+                                                "></select>
+                                            </div>
+                                            <div data-bind="visible: filter.selectedValue.doc_type() == 'StaticDatespanFilter'">
+                                                <select data-bind="
+                                                    options: report.filterConfig.date_range_options,
+                                                    value: filter.selectedValue.date_range
+                                                "></select>
+                                            </div>
+                                            <div data-bind="visible: filter.selectedValue.doc_type() == 'CustomDataAutoFilter'">
+                                                <input type="text" data-bind="value: filter.selectedValue.custom_data_property"/>
+                                            </div>
+                                            <div data-bind="visible: filter.selectedValue.doc_type() == 'StaticChoiceListFilter'">
+                                                <input type="text"
+                                                       class="choice_filter"
+                                                       style="width:300px"
```

Comment:
Why is that a problem on this page and not for other select2s? (Wondering if this style should be somewhere more general.)